src/functions.cpp

```diff
@@ -1,10 +1,9 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
-#include <algorithm>
+#include "functions.hpp"
 #include <iostream>
-#include <limits>
+#include <algorithm>
 
 void print(std::vector<int> const & numbers) {
-    for (int i = 0; i < numbers.size(); ++i) {
+    for (unsigned int i = 0; i < numbers.size(); ++i) {
```

Comment:
Read about std::size_t (instead of unsigned int).

Reply:
got it and going to change it. Thanks again